content/intermediate/operations/stack-policy-and-prevention-controls/index.md

```diff
@@ -53,11 +53,64 @@ Resources:
 
 In this next step, you will use the AWS CloudFormation Console to create a stack using the `stack-policy-lab.yaml` template file. Follow steps shown next:
 
+   :::::tabs{variant="container"}
+	::::tab{id="cloud9" label="Cloud9"}
+    Create a stack by following these steps:
+    1. Create a new JSON file for the stack policy.
+        
+        :::code{language=shell showLineNumbers=false showCopyAction=true}
+        touch policy-body.json
+        :::
+    1. Open this file in Cloud9 editor and paste in the following JSON code:
+    ```json
```

Comment:
Numbering after this JSON block was wrong until I updated to use `:::code` block
```suggestion
    :::code{language=json showLineNumbers=true showCopyAction=true}
```